index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Daniels Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <ul class="buger">
+                <a href="#"><li class="Mi">Daniel</li></a>                
+               <li class="ham">
+                <i class="fa fa-bars"></i> 
+               </li>  
+            </ul>
+        </nav>
+    </header>
+    <section class="homesection">
+        <div>
+<h1 class="intro">I'm Daniel glad to see you</h1>
+<h2 class="intro1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</h2>
```

Comment:
- [ ] [Optional] You should stick to the design as much as possible I would suggest you please adjust the spacing by looking at the Figma design. The logo should be on the right of the page as there is more gap. Please fix it and If you see the Figma design then you will find equal spacing from left and right but you have no spacing from left. Please fix it.
### Your design
![his](https://github.com/Danny-imo/my-project/assets/81727988/7ca25a2f-85f1-48ba-9deb-8d363af455b1)

### Figma design
![fimga](https://github.com/Danny-imo/my-project/assets/81727988/c4dcf58b-3222-493a-8f3e-d8c1aa579293)